Google.Api.Generator.Tests/ProtoTests/PublishingSettings/ServiceConfig.yaml

```diff
@@ -10,6 +10,7 @@ apis:
 
 publishing:
   librarySettings:
+  - version: testing.publishingsettings.wrong1
```

Comment:
I don't understand this test change. Or maybe I am not very familiar with the schema.

From this change it seems like the same library settings may apply to different versions? Like Beta1, Beta2, etc.?

What I was expecting from the old code comment:

```csharp
// Alternatively, ensure that each *published* service config only contains a single LibrarySettings entry.
```
is to have seen a new `librarySettings` property with different settings and a different version, to make sure we grab the one for this library.

Reply:
Oh, just me confused by yaml. This is doing exactly what I was expecting.